backend/src/services/v2/review.ts

```diff
@@ -188,6 +188,18 @@ export async function sendReviewResponseNotification(review: ReviewDoc, user: Us
   }
 }
 
+export async function getApprovedAccessRequestReviews(accessRequestIds: string[]) {
```

Comment:
This isn't really `getApprovedAccessRequestReviews`.  `get` implies you're returning something.  Instead I'd name this `checkAccessRequestsApproved`.